### PR TITLE
Replace SqlView with IcebergView

### PR DESCRIPTION
**Pull request by nastra**

### CODE REVIEW

File: servers/store/src/main/proto/table.proto [hunk 1]
```diff
@@ -41,9 +41,12 @@ message IcebergRefState {
   int32 sort_order_id = 4;
 }
 
-message SqlView {
+message IcebergView {
```
Comment (snazy) [inline servers/store/src/main/proto/table.proto:44]: IIUC Iceberg views correctly, `versionId` is similar to `snapshotId` in Iceberg tables and
Iceberg views have multiple versions of a view, analogous to multiple snapshot in Iceberg tables.

I also suspect that there should always be just one current "view-metadata-json" (analogous to one current "table-metadata-jsoin").

So I think you have to add glolbal state to Iceberg views like global state for Iceberg tables - i.e. store the current "view-metadata-json" via `IcebergMetadataPointer`.

/cc @rymurr

File: servers/store/src/main/java/org/projectnessie/server/store/TableCommitMetaStoreWorker.java [hunk 2]
```diff
@@ -68,11 +67,13 @@ public ByteString toStoreOnReferenceState(Content content) {
       }
       builder.setDeltaLakeTable(table);
 
-    } else if (content instanceof SqlView) {
-      SqlView view = (SqlView) content;
-      builder.setSqlView(
-          ObjectTypes.SqlView.newBuilder()
-              .setDialect(view.getDialect().name())
+    } else if (content instanceof IcebergView) {
```
Comment (snazy) [inline servers/store/src/main/java/org/projectnessie/server/store/TableCommitMetaStoreWorker.java:70]: Can you move that next to `IcebergTable` ?

File: ui/src/Explore/Components/ContentView.tsx [hunk 3]
```diff
@@ -104,15 +109,34 @@ const ContentView = (props: { tableContent: Content }): React.ReactElement => {
...
             <span>Type: </span>
             <span className={"ml-2"}> {tableContent.type}</span>
           </div>
+          {metadataLocation && (
+            <div>
+              <span>Metadata Location:</span>
+              <span className={"ml-2"}>{metadataLocation}</span>
+            </div>
+          )}
+          {versionId && (
```
Comment (snazy) [inline ui/src/Explore/Components/ContentView.tsx:128]: We don't show "details" for `IcebergTable` (above). Either for both of for none ;)

File: ui/src/Explore/Components/ContentView.test.tsx [hunk 4]
```diff
@@ -112,11 +112,14 @@ const detlaLakeContent: Content = {
   lastCheckpoint: "/path/to/delta/_delta_log/_last_checkpoint",
 };
 
-const sqlViewContent: Content = {
-  type: "VIEW",
+const icebergViewContent: Content = {
```
Comment (snazy) [inline ui/src/Explore/Components/ContentView.test.tsx:115]: Nit: move next to `icebergTableContent`

File: tools/content-generator/README.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ Generate commits
                                commit.
       --type=<contentType>   Content-types to generate. Defaults to
                                ICEBERG_TABLE. Possible values: ICEBERG_TABLE,
-                               DELTA_LAKE_TABLE, VIEW
+                               DELTA_LAKE_TABLE, ICEBERG_VIEW
```
Comment (snazy) [inline tools/content-generator/README.md:43]: Nit: reverse order (iceberg before dl)

File: README.md [hunk 6]
```diff
@@ -8,11 +8,11 @@
 
 Project Nessie is a Transactional Catalog for Data Lakes with Git-like semantics.
 
-Nessie supports Iceberg Tables, Delta Lake Tables and Sql Views.
+Nessie supports Iceberg Tables, Delta Lake Tables and Iceberg Views.
```
Comment (snazy) [inline README.md:11]: Nit: "... Iceberg Tables and Views, Delta Late Tables."

Comment (snazy) [reply README.md:11]: Maybe strike the whole sentence, it's redundant to line #15

File: README.md [hunk 7]
```diff
@@ -8,11 +8,11 @@
...
 
 More information can be found at [projectnessie.org](https://projectnessie.org/).
 
-Nessie supports Iceberg Tables, Delta Lake Tables and Sql Views. Additionally, Nessie is focused on working with the widest range of tools possible, which can be seen in the [feature matrix](https://projectnessie.org/tools/#feature-matrix).
+Nessie supports Iceberg Tables, Delta Lake Tables and Iceberg Views. Additionally, Nessie is focused on working with the widest range of tools possible, which can be seen in the [feature matrix](https://projectnessie.org/tools/#feature-matrix).
```
Comment (snazy) [inline README.md:15]: Nit: "... Iceberg Tables and Views, Delta Late Tables."

File: python/README.rst [hunk 8]
```diff
@@ -2,7 +2,7 @@
 Python API and CLI for Nessie
 =============================
 
-Nessie is a system to provide Git like capability for Iceberg Tables, Delta Lake Tables, Hive Tables and Sql Views.
+Nessie is a system to provide Git like capability for Iceberg Tables, Delta Lake Tables, Hive Tables and Iceberg Views.
```
Comment (snazy) [inline python/README.rst:6]: "Nessie is a system to provide Git like capability for Iceberg Tables and Views and Delta Lake Tables."

File: python/docs/content_list.rst [hunk 9]
```diff
@@ -1,39 +1,35 @@
 .. code-block:: bash
 
    Usage: nessie content list [OPTIONS]
-
+
```
Comment (snazy) [inline python/docs/content_list.rst:4]: Any idea why it's causing a diff in this file and log.rst?

**Review by snazy — COMMENTED** [review]
LGTM - but can you maybe reduce the amount of changed files underneath `python/`?

File: python/docs/content_list.rst (hunk 9 above)
Comment (nastra) [reply python/docs/content_list.rst:4]: I believe this happens when you run tox locally

**Review by snazy — DISMISSED** [review]

**Pull request merged**